libbeat/docs/outputconfig.asciidoc

```diff
@@ -521,6 +544,18 @@ The maximum number of files to save under <<path>>. When this number of files is
 oldest file is deleted, and the rest of the files are shifted from last to first. The default
 is 7 files.
 
+===== bulk_max_size
+
+The maximum number of events to bulk into batches. If the Beat sends single events, the events
+are collected into batches. If the Beat publishes a large batch of events (larger than the value
+specified by `bulk_max_size`), the batch is split.
+
+Specifying a larger batch size can improve performance by lowering the overhead of sending events. 
+However big batch sizes can also increase processing times.
+
+The default is 200.
```

Comment:
default is 2048

With bulk_max_size being some internal mechanism, redis plugin is not really aware of it. But redis output itself sends single events one by one only. 

Still comment is kinda correct, as batching can improve in-application performance.